tests/integration/plusminus/requirements_dev.txt

```diff
@@ -1,13 +1,13 @@
 pip
-bump2version==0.5.11
-wheel==0.33.6
-watchdog==0.9.0
-flake8==3.7.8
-tox==3.14.0
-coverage==4.5.4
-Sphinx==3.5.2
-twine==1.14.0
-sphinx_rtd_theme==0.5.1
+bump2version
```

Comment:
why did you remove the version numbers? does this work on multiple OS/platforms?

Reply:
Added the versions back.